### DISCUSSION TITLE
Should I get a formula with an at (`@`) character in front of it?

### DISCUSSION THREAD

**Discussion by verlok** [exceljs/exceljs#2005] [Q&A]
Hey there
I'm using the following code to create a shared formula in the excel file I'm generating

```js
  worksheet.fillFormula(
    `J2:J${lastRowNumber}`,
    'IFS(I2<0.9, "POOR! (--)", I2<1, "(-)", I2=1, "OK", I2>1.2, "BIG (++)", I2>1, "(+)")'
  );
```

But then, in the resulting excel, I get the formula with an at `@` character in front of it.
And so I get a `#NAME` error in the cell.

See screenshot here:

<img width="499" alt="Screenshot 2022-03-14 at 16 14 47" src="https://user-images.githubusercontent.com/1127721/158202620-6ce01876-ab42-49b1-b654-07a9fd984a89.png">

Why do I get that `@` and how can I get rid of it?

**Comment by yongzhangf** [comment]
A solution as below
[exceljs issues](https://github.com/exceljs/exceljs/issues/2006)
